orion/sites/admin/src/location/location.js

```diff
@@ -20,29 +20,79 @@ export class LocationPage extends React.Component {
         });
     }
 
+    onClickSelectAll() {
+        var items = document.getElementsByName("unpublished");
+        for (var i = 0; i < items.length; i++) {
+            if (items[i].type == "checkbox") {
+                items[i].checked = true;
+            }
+        }
```

Comment:
When a user selects all, maybe here is the time to modify this.state.numSelected;